protocol/testutil/app/block_advancement.go

```diff
@@ -49,10 +49,10 @@ func (b BlockAdvancementWithError) AdvanceToBlock(
 		) (haltchain bool) {
 			expectedError, found := b.ExpectedDeliverTxErrors[txIndex]
 			if found && expectedError != "" {
-				require.True(t, response.IsErr())
+				require.Conditionf(t, response.IsErr, "Expected CheckTx to error. Response: %+v", response)
```

Comment:
For my understanding why is `Conditionf` preferred over using `True` + calling the function?

Reply:
I'll just change to use `require.True` again